mpas_analysis/ocean/ocean_modelvsobs.py

```diff
@@ -31,7 +28,13 @@
 
 from ..shared.timekeeping.utility import get_simulation_start_time
 
-from ..shared.climatology import climatology
+from ..shared.climatology import get_lat_lon_comarison_descriptor, \
+    get_remapper, get_mpas_climatology_file_names, \
+    get_observation_climatology_file_names, \
+    compute_climatology, cache_climatologies, update_start_end_year, \
+    remap_and_write_climatology
```

Comment:
Following @pwolfram's earlier request, I am importing individual function names instead of the module.